uisdk/src/main/res/drawable/uisdk_background_light_grey_dashed_rounded.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <solid android:color="@color/kh_uisdk_background_light_grey" />
+    <solid android:color="@color/kh_uisdk_background2" />
```

Comment:
Light color of background2 is pure white. May want to use background1 instead given the deleted light_grey color.